app/views/layouts/_header.html.slim

```diff
@@ -8,6 +8,8 @@ header
         li = application_disambiguation_link
       - if current_user.try :current_student?
         li = link_to 'Status Updates', [:students, :status_updates], class: params[:controller] == 'students/status_updates' ? 'active' : ''
+      - if current_user.roles.includes?('supervisor')
```

Comment:
I would leave it. No reason for a separate helper here. Maybe the current_user.try syntax is a bit more efficient, and is more consistent within the header code. But either way, it is no big deal. YOLO